libgadget/fof.c

```diff
@@ -434,26 +481,23 @@ fof_primary_ngbiter(TreeWalkQueryFOF * I,
     }
     int other = iter->base.other;
 
-/* #pragma omp critical (_fofp_merge_) */
+    if(lv->mode == 0) {
```

Comment:
Nothing changes here other than indentation?

Reply:
That's right